app/src/api/auth/postRequest.ts

```diff
@@ -0,0 +1,42 @@
+export interface RequestBody {
+    username : string;
+    password : string;
+}
+
+const headers  = new Headers();
+headers.append('Content-type', 'application/json');
+
+export const requestOptions: RequestInit = {
+    method: 'POST',
+    headers: headers
+}
+
+export const sendPostRequest = async (url: string, requestOptions: RequestInit): Promise<object> => {
```

Comment:
Nice idea to have a request handler, I think it can also be improved by allow for more than just POST requests, e.g. GET, PUT, etc.

Reply:
you can specify the fetch type in requestOptions